tools/ttnn-standalone/ttnn-standalone.cpp

```diff
@@ -6,53 +6,61 @@
 
 // To generate forward function, run:
 // ./build/bin/ttmlir-opt --ttir-load-system-desc --ttir-implicit-device
-// --ttir-layout --convert-ttir-to-ttnn --convert-ttnn-to-emitc
-// test/ttmlir/Dialect/TTNN/simple_multiply.mlir | ./build/bin/ttmlir-translate
+// --ttir-layout --convert-ttir-to-ttnn --ttnn-decompose-layouts
+// --ttnn-deallocate --convert-ttnn-to-emitc
+// test/ttmlir/Silicon/TTNN/emitc/simple_add.mlir | ./build/bin/ttmlir-translate
 // -mlir-to-cpp -allow-unregistered-dialect
```

Comment:
is dash missing here `--mlir-to-cpp` ?

Reply:
In the official docs, it's with a single dash: https://mlir.llvm.org/docs/Dialects/EmitC/

If you do `./build/bin/ttmlir-translate --help`, it's with two.

I'll change it to two dashes next time, to align with optimizer, don't want to trigger CI for just this.